src/Model/__tests__/Model.test.ts

```diff
@@ -197,6 +203,17 @@ test('class instantiation Model should be updatable', async () => {
   expect(shouldHaveNewPassword?.password).toBe('root');
 });
 
+test('find one returns should not be dirty', async () => {
+  const username = nextUsername();
+  await User.create({
+    username,
+    password: 'rootroot',
+  });
```

Comment:
Can you also check that the value returned by `User.create()` is not dirty. Can be done in a separate test if you want.